log/src/test/java/io/airlift/log/TestLogger.java

```diff
@@ -201,6 +201,7 @@ public void testErrorWithThrowableNoMessageShortCircuit()
     }
 
     @Test
+    @SuppressWarnings("FormatStringAnnotation")
```

Comment:
See earlier comments about suppressions

Reply:
This is a valid case - we are tested that invalid log formats are reported - this is exactly why error prone is reporting this as well